vpr/src/analytical_place/flat_placement_mass_calculator.h

```diff
@@ -4,10 +4,22 @@
  * @author  Alex Singer
  * @date    February 2024
  * @brief   Mass calculation for AP blocks and logical/physical block/tile types
+ *
+ * This class is used to abstract away the concept of mass in the partial legalizer.
```

Comment:
I think this comment could be expanded to clarify what exactly is meant by "mass." I suggest adding an example, for instance, explaining that a molecule might consist of one LUT and one FF, and could be mapped to a cluster containing n LUTs and FFs. That would help solidify the meaning of "mass" in this context.